app/assets/stylesheets/pages/dashboard.scss

```diff
@@ -10,4 +10,60 @@
     background-repeat: no-repeat;
     background-position: right 10px center;
   }
+
+  .capacity {
+    table {
+      border-collapse: collapse;
+      border-spacing: 0;
+      width: 100%;
+
+      tr th {
+        font-weight: normal;
+        color: #757575;
+        text-align: left;
+        padding: 12px 0;
```

Comment:
Our standard at some point was to try to use rem over px

Reply:
Done - all instances of px replaced with rem.